### PR TITLE
document how to report issues

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
I created a template, not sure it is the right location though, can you please advise otherwise?

File: doc/.github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Delete this sentence after you read the "Issues" paragraph from the
+"guidelines for contributing" link above.
+
+If you are reporting a bug, please try to fill in the following, otherwise
+remove it along with this sentence.
+
+| Question       | Answer
+|----------------|-------------------------------------------------------------
+| Bundle version | composer show --installed | grep sonata-project/admin-bundle
```
Comment (Soullivaneuh) [inline doc/.github/ISSUE_TEMPLATE.md]: `--installed` is deprecated since composer 1.0. This is now the default behaviour.

Comment (Soullivaneuh) [inline doc/.github/ISSUE_TEMPLATE.md]: `sonata-project/*`

File: doc/.github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| Symfony version| composer show --installed | grep symfony/symfony
```
Comment (Soullivaneuh) [inline doc/.github/ISSUE_TEMPLATE.md]: `symfony/*`

**Comment by Soullivaneuh** [comment]
Could you add a message like "Check if you are up to date"?

File: doc/.github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (greg0ire) [inline doc/.github/ISSUE_TEMPLATE.md]: > `--installed` is deprecated since composer 1.0. This is now the default behaviour.

TIL

**Comment by greg0ire** [comment]
@Soullivaneuh done

File: doc/.github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Delete this sentence after you read the "Issues" paragraph from the
+"guidelines for contributing" link above.
+
+If you are reporting a bug, please try to fill in the following, otherwise
+remove it along with this sentence.
+
+| Question       | Answer
+|----------------|-------------------------------------------------
+| Bundle version | composer show | grep sonata-project/*
```
Comment (Soullivaneuh) [inline doc/.github/ISSUE_TEMPLATE.md]: `composer show sonata-project/*` :stuck_out_tongue:

File: doc/.github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+```
+Error message goes here
+```
+
+# Steps to reproduce
+
+# Expected results
+
+# Actual results
+``
```
Comment (Soullivaneuh) [inline doc/.github/ISSUE_TEMPLATE.md]: What is this? Oo

File: doc/CONTRIBUTING.md [hunk 5]
```diff
@@ -4,10 +4,46 @@ Thanks for you interest onto Sonata projects!
...
+Github by following these points are taken care of:
+
+* Check if the bug is not already reported!
+* A clear title to sum up the issue
+* A description of the workflow needed to reproduce the bug. Please try to make
+  sentence, dumping an error message by itself is not great.
+* If your issue is an error page, you must provide us with a stack trace.  With
+  recent versions of Symfony, you can even get stack traces as plain text at the
+end of the page. Just look for "Stack Trace (Plain Text)", and copy/paste what
+you see. **Do not** make a screenshot of the stack trace, as screenshots are
```
Comment (Soullivaneuh) [inline doc/CONTRIBUTING.md]: I think you can separate this on a second point. This is quite condensed now..

File: doc/.github/ISSUE_TEMPLATE.md (hunk 4 above)
Comment (greg0ire) [inline doc/.github/ISSUE_TEMPLATE.md]: A copy paste mistake :P

File: doc/CONTRIBUTING.md (hunk 5 above)
Comment (greg0ire) [inline doc/CONTRIBUTING.md]: Ok, I'll do that.

Comment (greg0ire) [inline doc/CONTRIBUTING.md]: ping @Soullivaneuh

File: doc/CONTRIBUTING.md [hunk 6]
```diff
@@ -4,10 +4,47 @@ Thanks for you interest onto Sonata projects!
...
+## Issues
+
+First, check if you are up to date: is your version still supported, and are
+you using the latest patch version?
+
+If you are not sure this is a bug, consider posting your question on [Stack
+Overflow](http://stackoverflow.com), using one of the sonata tags.
+If you happen to find a bug, we kindly request you to report it. However,
+before submitting it, please check the [project documentation available
+online](https://sonata-project.org/bundles/)
```
Comment (Soullivaneuh) [inline doc/CONTRIBUTING.md]: Missing a dot at the end.

Comment (greg0ire) [inline doc/CONTRIBUTING.md]: done @Soullivaneuh

**Comment by Soullivaneuh** [comment]
Let's merge this for now. Another contributions would be welcomed. :+1:

Thanks @greg0ire!

**Pull request merged**